packages/fx-core/src/core/middleware/utils/v3MigrationUtils.ts

```diff
@@ -126,28 +126,33 @@ export function getDownloadLinkByVersionAndPlatform(version: string, platform: P
 }
 
 export function outputCancelMessage(version: string, platform: Platform): void {
-  TOOLS?.logProvider.warning(`[core] Upgrade cancelled.`);
+  TOOLS?.logProvider.warning(`Upgrade cancelled.`);
   const link = getDownloadLinkByVersionAndPlatform(version, platform);
   if (platform === Platform.VSCode) {
     TOOLS?.logProvider.warning(
-      `[core] Notice upgrade to new configuration files is a must-have to continue to use current version Teams Toolkit. If you want to upgrade, please run command (Teams: Upgrade project) or click the “Upgrade project” button on tree view to trigger the upgrade.`
+      `Notice upgrade to new configuration files is a must-have to continue to use current version Teams Toolkit. Learn more at ${MetadataV3.v3UpgradeWikiLink}.`
     );
     TOOLS?.logProvider.warning(
-      `[core]If you are not ready to upgrade and want to continue to use the old version Teams Toolkit ${MetadataV2.platformVersion[platform]}, please find it in ${link} and install it.`
+      `If you want to upgrade, please run command (Teams: Upgrade project) or click the “Upgrade project” button on Teams Toolkit sidebar to trigger the upgrade.`
```

Comment:
The quote here is strange, please change to `"`.

Reply:
done